pkg/querier/queryrange/split_by_interval_test.go

```diff
@@ -440,6 +440,377 @@ func Test_splitByInterval_Do(t *testing.T) {
 	}
 }
 
+func Test_splitByInterval_Do_aligned(t *testing.T) {
+	ctx := user.InjectOrgID(context.Background(), "1")
```

Comment:
Can we instead assert may be called downstream with their request using mocks ?

Reply:
I think relying on response to verify request parameter makes the test hard to follow. Not sure if I'm being clear.